src/main/java/MainClass.java

```diff
@@ -0,0 +1,44 @@
+import java.util.Random;
+
+public class MainClass {
+    public static void main(String[] args){
+        int amountOfFigures = 10;
+        for (Figure figure: generate(amountOfFigures)) {
+            System.out.println(figure.draw());
+        }
+    }
+
+    public static Figure[] generate(int arraySize) {
+        Figure[] figures = new Figure[arraySize];
+        Random random = new Random();
+        for (int i = 0; i < arraySize; i++) {
+            switch ((int) ((Math.random() * (4 - 0)) + 0)) {
```

Comment:
```suggestion
            switch ((int) (Math.random() * 4)) {
```
на майбутнє, корисно виносити всі значення, як константи. Такі не зрозумілі числа називають: magic numbers
 Наприклад: 
private static final int COUNT_OF_SHAPE_TYPE = 4; чи щось подібне)